Slavahaim/hometask-2/get_middle_charachter.rb

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+def get_middle(str)
+  str[(str.size - 1) / 2..str.size / 2]
+end
+
+def run_cli
+  loop do
+    print 'Enter string: '
+    str = gets.strip
+    exit if str == 'exit!'
```

Comment:
We can extract `exit!` to a constant for example `EXIT_STATE`
We can put this constant at the beginning of the script